src/specifications/profile-resolution/profile-resolution-specml.xml

```diff
@@ -549,11 +549,7 @@ include-controls:
         </section>
         <section id="include-parent-controls">
           <head>with-parent-controls</head>
-          <p>Although similar to the above
-            <src>with-child-controls</src>, the optional
-            <src>with-parent-controls</src> applies to parents of the included control, and has the opposite default behavior. In order to maintain the structure of the source catalog, profile resolution includes all parents of an included control by default. If a profile author wants to change this structure, they should use an exclude directive that lists all of the undesired parents. As a shortcut for this,
-            <src>with-parent-controls</src> provides the following functionality:
-          </p>
+          <p>The the optional <src>with-parent-controls</src> flag applies to parents of the included control. If a profile author wants to change this structure, they should use an exclude directive that lists all of the undesired parents. <src>with-parent-controls</src> appears as a child object under a given inclusion directive and defines the behaviors listed below.</p>
```

Comment:
```suggestion
<p>The the optional <src>with-parent-controls</src> flag defines behavior applicable to parents or ancestors (a parent's parent etc) of the included control, in cases of nested controls. When it is left unstated or given the value <code>yes</code>, a control, when selected, brings its parent control(s) up to the root (group wrapper or catalog). Set to <code>no</code>, controls are to be decoupled from their parents, if any. <src>with-parent-controls</src> appears as a child object under a given inclusion directive and defines the behaviors listed below. Its semantics reflect the assumption that in most control catalogs, nesting of controls indicates logical dependencies that should normally be carried into result catalogs, unless given specific indication to the contrary.</p>
```

Reply:
I thought some of this is covered in the default behavior is explicitly covered in the lines below with a bullet by bullet breakdown (in for `with-child-controls` on `develop` the default `"no"` behavior is documented [here](https://github.com/usnistgov/OSCAL/blob/45ad64221aee3c7483ce88f66bb3956ba0b392dd/src/specifications/profile-resolution/profile-resolution-specml.xml#L545-L548) and for with `with-parent-controls` the default `"yes"` is [here](https://github.com/usnistgov/OSCAL/blob/45ad64221aee3c7483ce88f66bb3956ba0b392dd/src/specifications/profile-resolution/profile-resolution-specml.xml#L566-L569) on `develop` branch: ), so should we restate it?

There other suggestions seem to be substantive guidance on how parent and child controls work, should we not give them their own special treatment in separate work? (I am not opposed at all, just I feel this is important as I have asked a lot of questions in pairing that demonstrate I too can benefit from increased clarify on this topic, the edits may not even be enough.)

Reply:
Sure, anything that is already covered elsewhere can be cut!

Reply:
I also suggest we plan for an early complete walkthrough as soon as profile resolution unit testing is merged.

Mainly because the work there included not only tests themselves, but some very nice mechanics for integrating the spec document with the tests demonstrating the specs at the line item level.

Planning this would also alleviate the need to get this perfect right this minute - as long as it is not *wrong*, I would be okay with "only improved".

Reply:
```suggestion
          <p>The the optional <src>with-parent-controls</src> flag defines behavior applicable to parents or ancestors (a parent's parent etc) of the included control, in cases of nested controls. <src>with-parent-controls</src> appears as a child object under a given inclusion directive and defines the behaviors listed below. Its semantics assume that nesting of controls indicates logical dependencies in catalogs must persist unless a profile specifically indicates otherwise.</p>
```